code/modules/client/preference_setup/loadout/items/factions.dm

```diff
@@ -474,6 +476,14 @@
 	faction = "Private Military Contracting Group"
 	slot = slot_w_uniform
 
+/datum/gear/faction/grupo_amapola_officer
+ 	display_name = "grupo amapola officer cap"
+	path = /obj/item/clothing/head/misc/grupo_amapola
+	flags = GEAR_HAS_DESC_SELECTION
+	allowed_roles = list("Head of Security")
+	faction = "Private Military Contracting Group"
```

Comment:
The head of security cannot be part of the PMCG.

Reply:
stellar corporate conglomerate can take any faction gear